frappe/public/js/frappe/widgets/shortcut_widget.js

```diff
@@ -96,7 +96,7 @@ export default class ShortcutWidget extends Widget {
 		const label = get_label();
 		let color = this.color && count ? this.color.toLowerCase() : "gray";
 		$(
-			`<div class="indicator-pill no-indicator-dot ellipsis ${color}">${label}</div>`
+			`<div class="indicator-pill no-indicator-dot ellipsis ${color}">${__(label)}</div>`
```

Comment:
This does not work, since `label` already contains the count: "1 Available", "2 Available", "3 Available", etc. Translating this makes no sense, because we won't have translations for every possible number.

Reply:
The proper solution was already implemented here: https://github.com/frappe/frappe/pull/26169